Makefile

```diff
@@ -16,6 +16,8 @@ HELM_TEST ?= 1
 TESTS ?=
 # EXTRA_PROVE_ARGS: Additional prove arguments to pass
 EXTRA_PROVE_ARGS ?=
+# PROVE_COMMAND: If you want to use a different command than prove
+PROVE_COMMAND ?= prove
```

Comment:
So by default the new feature is not used, right? It would only be enabled in CI?
Could you add a sentence to the README on how to enable it?

Reply:
Yes, good idea :)